wdio.conf.js

```diff
@@ -0,0 +1,120 @@
+const fs = require('fs-extra')
+const path = require('path')
+const ip = require('ip')
+
+if (!process.env.BROWSERSTACK_USERNAME || !process.env.BROWSERSTACK_ACCESS_KEY) {
+  throw new Error('You must specify both BROWSERSTACK_USERNAME and BROWSERSTACK_ACCESS_KEY')
+}
+
+const TEST_SERVER_HOST = process.env.TEST_SERVER_HOST || ip.address()
+const TEST_SERVER_PORT = process.env.TEST_SERVER_PORT || 3000
+const REPORT_OUTPUT_DIR = './reports/browserstack'
+fs.mkdirpSync(path.resolve(REPORT_OUTPUT_DIR))
+
+exports.config = {
+  user: process.env.BROWSERSTACK_USERNAME,
+  key: process.env.BROWSERSTACK_ACCESS_KEY,
+  services: ['browserstack'],
+  browserstackLocal: true,
+
+  framework: 'jasmine',
+  specs: [
+    './tests/**/*.spec.js',
+  ],
+  suites: {
+    smoke: [
+      './tests/initial-load.spec.js',
+    ],
+  },
+
+  maxInstances: 1,
+  capabilities: [
+    // {
+    //   device: 'iPad 6th',
```

Comment:
is needed to add some Android tablets (UC Browser etc)

Reply:
Fixed